tableauserverclient/models/task_item.py

```diff
@@ -28,11 +30,25 @@ def from_response(cls, xml):
     @classmethod
     def _parse_element(cls, element):
         schedule = None
+        target = None
         schedule_element = element.find('.//t:schedule', namespaces=NAMESPACE)
+        workbook_element = element.find('.//t:workbook', namespaces=NAMESPACE)
+        datasource_element = element.find('.//t:datasource', namespaces=NAMESPACE)
         if schedule_element is not None:
             schedule = schedule_element.get('id', None)
+
+        # according to the Tableau Server REST API documentation,
+        # there should be only one of workbook or datasource
+        if workbook_element is not None:
+            workbook_id = workbook_element.get('id', None)
+            target = Target(workbook_id, "workbook")
+        if datasource_element is not None:
+            datasource_id = datasource_element.get('id', None)
+            target = Target(datasource_id, "datasource")
+        #
```

Comment:
Remove this extra '#' and the space around it (of course still making the style checker happy)